Improve Zulipbot messages

alya opened the pull request:
I generally tried to make the language Zulipbot uses clear but more succinct than it currently is, with links to documentation where relevant.

There are no logic changes, and all the variables used were already in the comment.

This PR also adds a checklist for brand new contributors, to encourage claiming the first issue only when ready. As described in the commit message, I think the way this is implemented here (with no logic changes) is not ideal. But we could try it and see how it goes.

I don't know how to test these changes.

timabbott commented on config/templates/needsReviewWarning.md:
```diff
@@ -1,3 +1 @@
-Hello @{assignees}, @{author} updated this pull request over {days} days ago in response to your review, but you haven't checked back in on their changes! Please leave some feedback for them or mark this pull request as ready for merging.
-
-Thank you for reviewing this pull request!
+Hello @{assignees}, @{author} updated this pull request over {days} days ago in response to your review, but you haven't checked back in on their changes! Please leave some feedback for them or mark this pull request as ready for merging. Thanks!
```
"ready for merging" feels off to me, maybe just "Please leave some feedback on next steps for this pull request"?

alya replied:
Sure.

timabbott replied:
WFM

andersk commented on config/templates/abandonWarning.md:
```diff
@@ -1,5 +1 @@
-You can reclaim this issue or claim any other issue by commenting `@{username} claim` on that issue.
-
-Thanks for your contributions, and hope to see you again soon!
+@{assignee} you have been unassigned from this issue because you have not made any updates for over {total} days. Please feel free to reclaim the issue if you decide to pick up again. Thanks!
```
This should be either “@{assignee}, you …” with a comma or “@{assignee} You …” with a capital letter. (Similarly for other strings.)

timabbott commented:
These changes all seem like improvements, thanks for doing this!

andersk commented on config/templates/contributorAddition.md:
```diff
@@ -1,10 +1,9 @@
-Welcome to Zulip, @{commenter}! We just sent you an invite to collaborate on this repository at https://github.com/{repoOwner}/{repoName}/invitations. Please accept this invite in order to claim this issue and begin a fun, rewarding experience contributing to Zulip!
+Welcome to Zulip, @{commenter}! To get started, please take the following steps **in order** and mark them off when done:
```
Arbitrary contributors won’t have permission to edit zulipbot’s comment, such as by checking its checkboxes.

alya replied:
Ah, good to know!

alya commented:
I pushed an update addressing all the feedback above.

We may want to hold off on merging the last commit (403ec2f) until we also have a fix to check for double assignments at the right time, as it's expected to make that possibility come up more often.

timabbott commented:
I merged the first 4 commits as the series ending with 36b0fe1c11342b3892945c330ec1cf7209e5df7c.

alya commented:
I was recently reminded of the text of our new contributor Zulipbot messages, and at this point I don't think it's worth waiting for technical changes before we make some improvements.

@timabbott would you be comfortable with merging the last commit on this PR? Or maybe we can tweak it a bit to specifically address double assignments, e.g.:

"You can contribute by providing feedback on their pull request." -> "You can unassign yourself and contribute by providing feedback on their pull request."

timabbott commented:
Yeah I think a tweak of that form is probably a reasonable option.
